code/drasil-lang/lib/Language/Drasil/Space.hs

```diff
@@ -86,9 +85,10 @@ getActorName :: Space -> String
 getActorName (Actor n) = n
 getActorName _         = error "getActorName called on non-actor space"
 
--- | Gets the inner 'Space' of a vector.
+-- | Gets the inner 'Space' of a vector or set.
 getInnerSpace :: Space -> Space
 getInnerSpace (Vect s) = s
+--getInnerSpace (Set s) = s
```

Comment:
Why is this commented out?

Reply:
I was testing in Space and must have forgot about this, it has been added back